lib/Runtime/Base/WindowsGlobalizationAdapter.h

```diff
@@ -142,6 +156,24 @@ namespace Js
         static HRESULT VerifyResult(HSTRING * result, HRESULT errCode);
 #endif
     };
-}
-#endif
+#endif // INTL_WINGLOB
+
+#ifdef INTL_ICU
```

Comment:
>INTL_ICU [](start = 7, length = 8)

Eventually, is there a plan to have `IntlAdapter` class from which we inherit `WindowsGlobalizationAdapter` and `ICUAdapter` ?